docs/initial-setup.md

```diff
@@ -0,0 +1,24 @@
+# Initial setup
+
+rules_apko requires a one-time setup to configure bazel to be able to make partial fetches.
```

Comment:
link to the upstream issue on bazelbuild/bazel so enraged users have somewhere useful to vent?

Reply:
already working on but leave it there for posterity.

Reply:
https://github.com/bazelbuild/bazel/issues/17829